Packs/Phishing/Scripts/PhishingDedupPreprocessingRule/PhishingDedupPreprocessingRule.py

```diff
@@ -247,10 +248,20 @@ def create_new_incident():
     return_entry('This incident is not a duplicate of an existing incident.')
 
 
-def create_new_incident_low_similarity(existing_incident, similarity):
-    message = 'This incident is not a duplicate of an existing incident.\n'
-    message += 'Most similar incident found is #{} with similarity of {:.1f}%.\n'.format(existing_incident['id'],
-                                                                                         similarity * 100)
+def format_similar_incident(incident, similairy):
+    return {'Id': "[%s](#/Details/%s)" % (incident['id'], incident['id']),
+            'Name': incident['name'],
+            'Closed': incident.get('closed') != "0001-01-01T00:00:00Z",
+            'Time': str(incident['created']),
```

Comment:
perhaps adding more friendly formatting for `created` timedate

Reply:
it's actually looks fine it's timestamp formatting